js/information.js

```diff
@@ -343,47 +343,49 @@ function updateSchedule() {
  * Sets current weather conditions
  */
 function setWeather() {
-  /**
-   * Stores current weather data
-   * @type {Object}
-   */
-  var weather;
-
-  // New promise
-  new Promise((resolve, reject) => {
-    /** @type {string} */
-    var apikey = localStorage.getItem("accuweather-api-key");
-    try {
-      // Gets current weather data from weather API
-      $.getJSON(
-        `https://dataservice.accuweather.com/currentconditions/v1/49569?apikey=${apikey}`,
-        function (wth) {
-          if (wth == undefined) {
-            reject(); // Reject promise if nothing returned
-          }
-          weather = wth[0];
-          resolve(); // Resolve promise if all succeeds
-        }
-      ).fail(() => {
-        reject(); // Reject promise if API request fails
-      });
-    } catch (err) {
-      reject(); // Reject promise if error is thrown
-    }
-  })
+  getWeather()
     .then(() => {
       // Sets weather data once the promise is resolved
-      $("#w-icon").show();
       $("#w-icon").attr("src", `img/weathericons/${weather.WeatherIcon}.svg`);
-      $("#temp").text(`${weather.Temperature.Metric.Value.toFixed(1)}°C`);
+      $("#w-icon").show();
+      $("#temp").text(
+        `${Math.round(weather.Temperature.Metric.Value.toFixed(1))}°C`
```

Comment:
Remove .toFixed(1), and where are you setting the value for weather?

Reply:
Should be fixed now. Thanks!